runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -399,6 +402,10 @@ private void updateConfig(int slot, GrandExchangeOffer offer)
 			savedOffer.setSpent(offer.getSpent());
 			savedOffer.setState(offer.getState());
 			setOffer(slot, savedOffer);
+			if (config.enableGELimitReset())
+			{
+				updateLimitTimer(offer);
```

Comment:
This should probably update regardless, and make the toggle affect the UI. That way you can toggle it on later and it already know what you have bought.

Reply:
Removed the if case so it triggers every time